src/main/java/com/bravo/user/dao/model/mapper/PaymentMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.bravo.user.dao.model.mapper;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.model.dto.PaymentDto;
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+@Mapper(componentModel = "spring")
```

Comment:
Can you explain what `componentModel = "spring"` means?

Reply:
@wheelerswebservices it means construct mapper as spring bean. Because mapper has different construction types, using factory, spring beans etc.